adsbibdesk.py

```diff
@@ -1615,24 +1644,34 @@ def parse(self, url):
         """
         try:
             # Detect and decode page's charset
-            logging.debug("Parsing MNRAS url %s" % url)
-            response = requests.get(url,
-                       headers={'User-Agent':
-                                'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_9_5) AppleWebKit/537.36 \
-                                (KHTML, like Gecko) Chrome/61.0.3163.100 Safari/537.36'})
-            self.feed(response.text)
-            #connection = urllib2.urlopen(url)
-            #encoding = connection.headers.getparam('charset')
-            #if encoding is not None:
-            #    logging.debug("Detected MNRAS encoding %s" % encoding)
-            #    page = connection.read().decode(encoding)
-            #    self.feed(page)
-            #else:
-            #    logging.debug("No detected MNRAS encoding")
-            #    page = connection.read()
-            #    self.feed(page)
+            logging.debug("Parsing EJOURNAL url %s" % url)
+            
+            #######
+            #   requests is not always reliable. (javascript page?)
```

Comment:
This is weird.  Maybe we should just have a `retry` loop here, rather than relying on the lower-level connectors?  But, for now, this is fine.

Reply:
If I remember correctly, I came across some occasions that certain journals rejected ```requests```. So I reverted back to ```urlopen```

Reply:
That's really weird.  I did encounter some of the same at one point, but I was able to work around it by spoofing a browser, e.g., as you have done here:
https://github.com/jonathansick/ads_bibdesk/pull/73/files#diff-b520f4c7117ebe6651f9357c2f779758R837

Does that solution not work for this case?

Reply:
It's really interesting that some journals would reject `requests` but accept `python`.  If you have any data on which do, it might be worth raising that information to the broader community, because it seems like an awful practice.

See https://docs.python.org/3.6/howto/urllib2.html#headers for what python does itself.

(anyway, whatever workaround you need here is fine by me, I just want to call out journals for bad practices if they're engaged in them)

Reply:
I did try that trick then, but it didn't work. I loosely remembered the journal was one of these three: sci, spie, or na. 
We can move back to ```requests``` if it was just a one-off event.

Reply:
Sure, I'll merge this PR now, but with the note-to-self that we might prefer to revert to the commented-out code if we can figure out how to hack around the bad journals.